source/manual/non-ha-node-classes.html.md

```diff
@@ -0,0 +1,25 @@
+---
+owner_slack: "#re-govuk"
+title: Node classes without redundancy
+layout: manual_layout
+parent: "/manual.html"
+section: Applications
+important: true
+last_reviewed_on: 2019-07-29
+review_in: 6 months
+---
+
+Below we provide a list of node classes that are not configured to be highly available (HA). This means, that short outages of these machines are architecturally considered acceptable. Recently, the installation of a critical software componenent on a non-redundant node class did cause an incident.
+
+The situation in question involved pgbouncer, required for all Postgresql database accesses at the time, to be installed on the db-admin machines. Because the architectural design anticipated machines of the db-admin node class to be used as jumpboxes for administrative database access only, a high load introduced on a database created a global Postgresql outage for GOV.UK.
```

Comment:
I'm not sure what `global Postgresql outage` means, over just a `Postgresql outage`?

Reply:
Fair - I think I was thinking along "all the databases"..